dbt/include/athena/macros/utils/timestamps.sql

```diff
@@ -1,5 +1,15 @@
+{%
+    pyathena converts time zoned timestamps to strings so lets avoid them now()
+%}
+
 {% macro athena__current_timestamp() -%}
-    -- pyathena converts time zoned timestamps to strings so lets avoid them
-    -- now()
     cast(now() as timestamp)
 {%- endmacro %}
+
+{%
+  Macro to get the end_of_time timestamp
+%}
+
+{% macro end_of_time() -%}
+  CAST('9999-01-01' AS timestamp)
+{%- endmacro %}
```

Comment:
shall we call it `athena__end_of_time`?

Reply:
I have change the macro name. But unfortunately I am running into an error when I add precision to the timestamp. I guess it doesn't work with v2?

Reply:
you're right, it doesn't work with v2, I think that either we use the version with precision or the one without, but either way we need to put a note in the readme that `athena__end_of_time` need to be ovewritten based on the engine version in use by the user.

Reply:
Updated readme as suggested. I wonder if there is a way to return timestamp or timestamp(3) depending on the engine version so that the user doesn't have to do manual overrides. 

Resolving this thread for now.